clang-tools-extra/test/clang-doc/json/class-requires.cpp

```diff
@@ -23,7 +23,6 @@ struct MyClass;
 // CHECK-NEXT:        "End": true,
 // CHECK-NEXT:        "Expression": "Addable<T>",
 // CHECK-NEXT:        "Name": "Addable",
-// CHECK-NEXT:        "Path": "",
```

Comment:
Which objects still have a `Path` key? Do any, besides the GlobalNamespace?

Reply:
Sounds good. lets document that someplace... maybe the JSON Generator? Almost feels like it belongs on the Info def, but that isn't really correct. We're getting to the point where we need real docs that describe the end to end design of the documentation generation.

Anyway, for now this is fine, lets just add a comment somewhere that describes this.